source/_includes/menus/aws.md

```diff
@@ -0,0 +1,45 @@
+`````{grid} 1 2 2 3
+:gutter: 2 2 2 2
+
+````{grid-item-card}
+:link: /cloud/aws/ec2
+:link-type: doc
+Elastic Compute Cloud (EC2)
+^^^
+Launch an EC2 instance and run RAPIDS.
+
+{bdg-primary}`single-node`
+````
+
+````{grid-item-card}
+:link: /cloud/aws/eks
+:link-type: doc
+Elastic Kubernetes Service (EKS)
+^^^
+Launch a RAPIDS cluster on managed Kubernetes.
+
+{bdg-primary}`multi-node`
```

Comment:
Curious on your thoughts of making `multi-node` a different color than `single-node`

Reply:
Yeah I think that would be great. I want to think a bit more deeply about tagging in the docs, especially as we integrate these docs with cloud-ml-examples more closely. So maybe we could have this discussion then?

Reply:
Sounds good.